test/unit/Query/QueryCollection.test.php

```diff
@@ -4,24 +4,71 @@
 use Gt\Database\Connection\DefaultSettings;
 use Gt\Database\Connection\Driver;
 use Gt\Database\Result\ResultSet;
+use PHPUnit_Framework_MockObject_MockObject;
 
 class QueryCollectionTest extends \PHPUnit_Framework_TestCase {
+/** @var  QueryCollection */
+private $SUT;
```

Comment:
It's a style question so your call.  It's easy for me to refactor but I won't do it unless you want it done.  Personally I like SUT (esp. if it's consistent throughout all test but that's not the case here obviously).